docs/user-guide/garbage-collection.md

```diff
@@ -0,0 +1,36 @@
+---
+assignees:
+- caesarxuchao
+
+---
+
+* TOC
+{:toc}
+
+## Garbage Collection
+
+Note: the Garbage Collection is a beta feature and is enabled by default in Kubernetes version 1.4.
```

Comment:
@caesarxuchao this doc may be a good place to list behavior differences with/without the garbage collector, too.